packages/aws-cdk/test/init.test.ts

```diff
@@ -88,3 +92,14 @@ async function withTempDir(cb: (dir: string) => void | Promise<any>) {
     await fs.remove(tmpDir);
   }
 }
+
+/**
+ * The init templates rely on parsing the current major version to find the correct template directory.
+ * During tests, the current package version is '0.0.0', rather than a specific version.
+ * The below mocks the versionNumber to return the same major version as the current release.
+ */
+function mockVersionNumber() {
+  // eslint-disable-next-line @typescript-eslint/no-require-imports
+  const releaseJson = require(`${__dirname}/../../../release.json`);
```

Comment:
Note - I don't _love_ this. I'm totally open to better ways to get the correct major version for the branch (if such a thing exists).

Reply:
can we instead run the tests for both v1 and v2? (maybe update `cliTest` so that there's no test case duplication?)
skimming through the tests, it seems it's just checking that specific files get created. I'm assuming it doesn't try to install dependencies, but maybe I'm wrong.

Reply:
> can we instead run the tests for both v1 and v2?

I don't think this makes sense. v1 (master/this change) obviously can't run the v2 templates. We can add tests to ensure the v2 CLI can instantiate the v1 templates, but I'm not sure what value that provides, exactly.

> I'm assuming it doesn't try to install dependencies, but maybe I'm wrong.

These tests do run `npm install` after initializing the template to prove the template is valid.